akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -703,6 +703,33 @@ final case class Referer(uri: Uri) extends jm.headers.Referer with RequestHeader
   def getUri: akka.http.javadsl.model.Uri = uri.asJava
 }
 
+//https://tools.ietf.org/html/rfc7231#section-7.1.3
+sealed trait `Retry-After` extends jm.headers.RetryAfter
+final case class RetryAfterDuration(delay: Long) extends `Retry-After` with ResponseHeader {
+  def renderValue[R <: Rendering](r: R): r.type = r ~~ delay
+
+  protected def companion = `Retry-After`
```

Comment:
is this "shared" reference to the same ModeledCompanion at risk of creating some problem later on?